frog/imports/api/activities.js

```diff
@@ -1,6 +1,6 @@
 import { Meteor } from 'meteor/meteor';
 import { Mongo } from 'meteor/mongo';
-import { uuid } from 'frog-utils';
+import uuid from 'frog-utils';
```

Comment:
It's an error. Must have happened when I was changing uuid for cuid, but then as we discussed it I changed the uuid export in frog-utils only.